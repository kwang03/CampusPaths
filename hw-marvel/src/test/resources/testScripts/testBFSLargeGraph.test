#Tests the BFS on a graph with 5 nodes

LoadGraph myGraph largeGraph.csv

FindPath myGraph Char1 Char2

FindPath myGraph Char1 Char4

FindPath myGraph Char1 Char3

FindPath myGraph Char1 Char5

FindPath myGraph Char1 Char6