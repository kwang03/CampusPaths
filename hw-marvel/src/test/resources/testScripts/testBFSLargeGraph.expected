#Tests the BFS on a graph with 5 nodes

loaded graph myGraph

path from Char1 to Char2:
Char1 to Char2 via Book1

path from Char1 to Char4:
Char1 to Char2 via Book1
Char2 to Char4 via Book2

path from Char1 to Char3:
no path found

path from Char1 to Char5:
Char1 to Char5 via Book1

unknown: Char6